Automattic/wp-calypso
# Extract wordpress externals 31351

#### Changes proposed in this Pull Request

- Moving most @WordPress external dependencies to the calypso-build package.
- babel-related dependencies stay in the main package.json file as they seem too interwoven and splitting them away requires more work than expected, so it requires a separate issue.

#### Testing instructions

1. Delete your local `./node_modules/` directory to uninstall all local dependenceis
2. Run `npm install` and then `npm start` and see if anything breaks.
3. Try building the blocks

Closes #31351

🍐 with @gititon
